packages/theme/src/__tests__/__snapshots__/theme.test.tsx.snap

```diff
@@ -26,7 +26,7 @@ Object {
     "colorAvatarBackgroundGreen": "#8AFECC",
     "colorAvatarBackgroundLightBlue": "#A9EBE8",
     "colorAvatarBackgroundOrange": "#FBBF24",
-    "colorAvatarBackgroundPink": "#FF9788",
+    "colorAvatarBackgroundPink": "#A9EBE8",
```

Comment:
Did you already change the token? I can see only the snapshot in this PR :thinking:

Reply:
It was changed in a different PR but the snapshot didn't fail on that one https://github.com/Localitos/pluto/pull/656 🤔